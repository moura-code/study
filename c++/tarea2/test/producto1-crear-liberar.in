crearFecha 1/1/2001
crearProducto 7 producto1 5
liberarProducto
crearFecha 1/1/2001
crearProducto 7 producto1 15
liberarProducto
Fin
