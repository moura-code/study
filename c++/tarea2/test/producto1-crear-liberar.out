Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Fecha creada en forma exitosa.
2> Producto creado en forma exitosa.
3> Producto liberado en forma exitosa.
4> Fecha creada en forma exitosa.
5> Producto creado en forma exitosa.
6> Producto liberado en forma exitosa.
7> Fin.
